makefile

```diff
@@ -22,6 +22,18 @@ ifeq ($(PLATFORM),x86_64)
     CXXFLAGS += -maes
 endif
 
+ifeq ($(PLATFORM),ppc64)
+VSX=$(shell $(CC) -mcpu=native -dM -E - < /dev/null | grep VSX )
```

Comment:
Is running this command necessary? What would happen if you always set `-mcpu=native -flax-vector-conversions` regardless of VSX support?

Also the GCC documentation says:
> **-flax-vector-conversions**
> Allow implicit conversions between vectors with differing numbers of elements and/or incompatible element types. This option should not be used for new code.

So it's best to remove the option and use explicit conversions.

Reply:
the VSX define is set by the cpu type, -flax-vector-conversions shoudn't change it.
Ok, I'll change the code to use explicit conversions.

Reply:
I recommend removing this line: `VSX=$(shell $(CC) -mcpu=native -dM -E - < /dev/null | grep VSX )`.

So the script should be:
```makefile
ifeq ($(PLATFORM),ppc64)
    CXXFLAGS += -mcpu=native
endif
```

Additionally, please also modify the CMakeLists.txt file accordingly. Merge #56 to avoid conflicts.

Reply:
Ok, merged #56 and fixed up the makefile.